src/static/templateUrl.test.ts

```diff
@@ -103,3 +103,35 @@ it.each([
 ])('combines %s, %s and %s into %s', (url, base, parameters, expected) => {
   expect(new TemplateURL(url, base, parameters).toString()).toBe(expected);
 });
+
+it.each([
+  ['../simple', 'https://example.com', {}, 'https://example.com/simple'],
```

Comment:
Can be add a test to include both templated url and path traversal, something like ../{organizationId}/test

Reply:
The requested test has been added